js/public/index.html

```diff
@@ -19,6 +19,10 @@
     Learn how to configure a non-root public URL by running `npm run build`.
   -->
   <title>React App</title>
+  <link rel="stylesheet" href="https://amptesting.sites.stanford.edu/core/misc/dropbutton/dropbutton.css"/>
```

Comment:
LOL. Perhaps use the default content site as that is likely to stick around longer than this one.

Reply:
yeah its just for local `webpack-dev-server` so if it breaks... :man_shrugging: